src/unistd.rs

```diff
@@ -356,6 +375,13 @@ pub fn getcwd() -> Result<PathBuf> {
     }
 }
 
+// According to the POSIX specification, -1 is used to indicate that
```

Comment:
I use this function in four different flavors of chown. Should I copied the comment to each of them? Why it does not make it sense here?